### ISSUE TITLE
Issue when nextjs trailingSlash = true

### ISSUE THREAD

**Issue by dylee-sparta** [edgestorejs/edgestore#35]
Background:

- I am currently developing a web frontend application using Next.js.
- The application is built on Next.js version 14.0.4.
- I am attempting to upload files using a storage service called Edgestore.
- To my understanding, Edgestore provides cloud storage services similar to AWS's S3.

Error Message:
I encountered an error while trying to integrate Edgestore into my project. The error message is as follows:
```
EdgeStoreError: Failed to parse response. Make sure the API is correctly configured at http://localhost:3000/api/edgestore/init/
```

Suspected Causes:

- I suspect this issue might be related to the version of Next.js I am using. Could there be any compatibility issues with Next.js 14.0.4?
- Or my layout.tsx 
- Here is my src/app/layout.tsx file
- Because when i debug my project code, i deleted all code related edgestore without <EdgeStoreProvider>
``` typescript
export default function RootLayout({
  children,
}: {
  children: React.ReactNode;
}) {
  return (
    <html lang="en">
      <body>
        <EdgeStoreProvider>{children}</EdgeStoreProvider>
        <Toaster />
      </body>
    </html>
  );
}
```

**Comment by perfectbase** [comment]
Hi @dylee-sparta!
There should be no compatibility issue with next 14.
Can you take a look at the [troubleshooting](https://edgestore.dev/docs/troubleshooting) docs? It should help you fix your problem. If you still have problems after it, feel free to reach out!

**Comment by perfectbase** [comment]
Oh! I wasn't aware of this problem. Thank you for reporting it. Let's leave the issue open for now until we handle that.

**Comment by perfectbase** [comment]
This should be fixed in version [0.2.1](https://github.com/edgestorejs/edgestore/releases/tag/v0.2.1).
